tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = python38,python39,python310,python311,pylint,flake8
+envlist = py38,py39,py310,py311,pylint,flake8
```

Comment:
When using names like `python*`, tox always uses the _default_ (current?) Python version; when using `py*`, it looks for appropriate Python versions (in `$PATH` or via `py.exe`, depending on OS).